session.go

```diff
@@ -144,6 +144,10 @@ func NewSession(cfg ClusterConfig) (*Session, error) {
 		return nil, errors.New("Can't use both Authenticator and AuthProvider in cluster config.")
 	}
 
+	if cfg.Consistency.IsSerial() {
+		return nil, fmt.Errorf("the default consistency level is not allowed to be SERIAL or LOCAL_SERIAL. Recived value: %v", cfg.Consistency)
```

Comment:
No, it's fine if `Consistency` has `SERIAL` or `LOCAL_SERIAL` (it won't work with writes but sure, maybe the app only has reads). I meant that `cfg.SerialConsistency` has to be `SERIAL` or `LOCAL_SERIAL`

Reply:
I agree, it slipped my mind that consistency could be SERIAL for read operation, thank you for noticing it.